Objects/dictobject.c

```diff
@@ -1686,50 +1686,12 @@ PyDict_GetItem(PyObject *op, PyObject *key)
 }
 
 Py_ssize_t
-_PyDict_GetItemHint(PyDictObject *mp, PyObject *key,
-                    Py_ssize_t hint, PyObject **value)
+_PyDict_GetItemSpecialize(PyDictObject *mp, PyObject *key, PyObject **value)
```

Comment:
the result placed in `value` is not used in the callers of `_PyDict_GetItemHint`. Could you remove the `value` parameter.

Reply:
Thanks for both suggestions!